netbox/dcim/forms/bulk_import.py

```diff
@@ -1245,18 +1266,40 @@ def _clean_side(self, side):
         assert side in 'ab', f"Invalid side designation: {side}"
 
         device = self.cleaned_data.get(f'side_{side}_device')
+        circuit = self.cleaned_data.get(f'side_{side}_circuit')
         content_type = self.cleaned_data.get(f'side_{side}_type')
         name = self.cleaned_data.get(f'side_{side}_name')
-        if not device or not content_type or not name:
+
+        if not (device or circuit) or not content_type or not name:
             return None
+        
+        if device and circuit:
+            raise forms.ValidationError(
+                _("Side {side_upper}: Both `device` and `circuit` cannot be specified at the same time").format(side_upper=side.upper())
+            )
 
         model = content_type.model_class()
         try:
-            if device.virtual_chassis and device.virtual_chassis.master == device and \
-                    model.objects.filter(device=device, name=name).count() == 0:
-                termination_object = model.objects.get(device__in=device.virtual_chassis.members.all(), name=name)
-            else:
-                termination_object = model.objects.get(device=device, name=name)
+            
+            # Should never happen as we return None above if we don't have a device or circuit
+            assert device or circuit
```

Comment:
One of the things we came up to in another PR, is asserts are not a guaranteed thing since it requires python to be compiled with debug=true.

I am actually going to defer to Jeremy on whether we want to stick with asserts or not.